### PR TITLE
Add https://tauri.localhost for Windows

### LINKED ISSUE #878: For a bit the Windows build have had rendering issues
Seem be stuck on the blur

image

### PR DESCRIPTION
**Pull request by pierremtb**
Fixes #878

### CODE REVIEW

**Review by Irev-Dev — APPROVED** [review]

**Comment by Irev-Dev** [comment]
Oh sorry, approval might have been premature, did this work?

**Comment by pierremtb** [comment]
@Irev-Dev Yes! Sorry created this before verifying (and Github autopopulated the misleading desc here)

With these changes plus a local revert of 3b15bc1 (that @franknoirot will try to fix today), the error is gone and I can get things to render

image

**Comment by pierremtb** [comment]
image
also good on macOS (stream problem tho)

File: src/lang/wasm.ts [hunk 1]
```diff
@@ -66,13 +66,16 @@ const initialise = async () => {
...
+      ? 'tauri://localhost' // custom protocol for macOS
+      : window.location.origin.includes('tauri.localhost')
+      ? 'https://tauri.localhost' // fallback for Windows
       : window.location.origin.includes('localhost')
       ? 'http://localhost:3000'
       : window.location.origin && window.location.origin !== 'null'
       ? window.location.origin
       : 'http://localhost:3000'
   const fullUrl = baseUrl + '/wasm_lib_bg.wasm'
+  console.log(`Full URL for WASM: ${fullUrl}`)
```
Comment (franknoirot) [inline src/lang/wasm.ts:78]: Could remove this but it's fine for now

**Review by franknoirot — APPROVED** [review]
I've confirmed on my Windows machine with the artifact from this PR that the fetch error is gone from the console, which I could still see in tests last night even though my unrelated path issue was also occurring.

**Comment by franknoirot** [comment]
I've just tested the artifact on Mac and got a working stream still. I'm going to merge this, so that my PR for fixing #881 can have both fixes in its artifact.

**Pull request merged**